src/Factory/Consignment/ConsignmentFactory.php

```diff
@@ -282,6 +282,14 @@ private function getEmailConfiguration(): string
         return $emailConfiguration ? $this->orderData['email'] : '';
     }
 
+    /**
+     * @return bool
+     */
+    public function isConceptFirstConfiguration(): bool
+    {
+        return Configuration::get(Constant::PROCESS_SHIPMENT_INSTANTLY);
```

Comment:
De constante is nu het tegenovergestelde (in tekst) van de werking ervan, dat is verwarrend... Ik zou die constante ook iets met CONCEPT_FIRST noemen